script.js

```diff
@@ -1,46 +1,221 @@
+const upKeys = document.querySelectorAll(".caseUp");
+const downKeys = document.querySelectorAll(".caseDown");
+
+upKeys.forEach((el) => el.classList.toggle("hidden"));
+
+function changeLang() {
+  let lang = localStorage.getItem("lang");
+  if (!lang) lang = "ru";
+  if (lang === "ru") {
+    enKeys.forEach((el) => el.classList.remove("hidden"));
+    ruKeys.forEach((el) => el.classList.add("hidden"));
+    localStorage.setItem("lang", "en");
+  } else {
+    enKeys.forEach((el) => el.classList.add("hidden"));
+    ruKeys.forEach((el) => el.classList.remove("hidden"));
+    localStorage.setItem("lang", "ru");
+  }
+}
+function hideLangKeys() {
+  let lang = localStorage.getItem("lang");
+  if (!lang) lang = "ru";
+  if (lang === "ru") {
+    enKeys.forEach((el) => el.classList.add("hidden"));
+  } else {
+    ruKeys.forEach((el) => el.classList.add("hidden"));
+  }
+}
+hideLangKeys();
+function caps() {
+  upKeys.forEach((el) => el.classList.toggle("hidden"));
+  downKeys.forEach((el) => el.classList.toggle("hidden"));
+}
+// event
+const keyControls = ["Backspace", "Tab", "CapsLock", "ShiftLeft", "ControlLeft", "MetaLeft",
+  "AltLeft", "Space", "AltRight", "ControlRight", "ShiftRight", "Enter", "Delete", "Backspace"];
+
+function printKey(elem) {
+  function findKey(el) {
+    for (let i = 0; i < el.children.length; i += 1) {
+      const str = el.children[i].classList.value;
+      if (str.indexOf("hidden") === -1) {
+        findKey(el.children[i]);
+      }
+    }
+    return elem.innerText;
+  }
+  let returnPointer;
+  if (keyControls.indexOf(elem.classList[1]) === -1) {
+    const char = findKey(elem);
+    screen.value += char;
+  } else {
+    switch (elem.classList[1]) {
+      case "Space":
+        screen.value += " ";
+        break;
+      case "Tab":
+        screen.value += "     ";
+        break;
+      case "Backspace":
+        returnPointer = screen.selectionEnd - 1;
+        screen.value = screen.value.substring(0, screen.selectionEnd - 1)
+          + screen.value.substring(screen.selectionEnd);
+        screen.selectionEnd = returnPointer;
+        break;
+      case "Delete":
+        returnPointer = screen.selectionEnd;
+        screen.value = screen.value.substring(0, screen.selectionEnd)
+          + screen.value.substring(screen.selectionEnd + 1);
+        screen.selectionEnd = returnPointer;
+        break;
+      case "Enter":
+        screen.value += "\n";
+        break;
+      // no default
     }
+  }
 }
+window.addEventListener("keydown", (event) => {
+  // console.log(event);
+  event.preventDefault();
+  const { code } = event;
+  if (event.ctrlKey && event.altKey) changeLang();// смена языка
+  if (code === "CapsLock") caps();
+  if (event.key === "Shift" && event.shiftKey && !event.repeat) caps();// Если зажат шифт
+  const elem = keyboard.querySelector(`.${code}`);
+  if (!elem) return false;
+  elem.classList.add("key__down");
+  printKey(elem);
+  return "keydown";
```

Comment:
You can rewrite your code as 
```javascript
  if (elem) {
    elem.classList.add("key__down");
    printKey(elem);
  }
```

to avoid unnecessary returns;